docs/approvals/service-now/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Service Now Integration
+description: Approvals, approvals, approvals
+position: 10
+---
+:::warning
+The ServiceNow Integration feature is only available as an EAP for version 2022.2.5556 or later and it requires a feature license to be able to use it.
+:::
+
+## Overview
+
+The Octopus Deploy/ServiceNow integration allows users to block the execution of specifically configured deployments unless they have a corresponding approved ServiceNow **Change Request** (CR).
+
+For configured project and environment combinations, Octopus Deploy will 
+
+To enable this behavior, both systems must be configured before deployments can be managed.
+
+| Project | Environment Change Controlled | Environment **Not** Change Controlled| 
+| --------|--|-- |
+| Change Controlled | _Approval Required_  | No Approval Required | 
+| **Not** Change Controlled | No Approval Required | No Approval Required |
+
+## Getting started
+
+The Service Now integration requires Octopus 2022.2.5556 or later, and an Octopus license with the Service Now Integration feature enabled.
+
+To get started:
+1. Configure ServiceNow
+1. Obtain a ServiceNow feature license for your Octopus instance 
+1. Configure a connection from Octopus to ServiceNow
```

Comment:
In the past, when writing Test docs, I tried to "scope" the statement, so people could drill into it (but maybe this is a higher level)
* In Octopus, configure a ServiceNow Connection using parameters from step 1
* In Octopus, configure projects and environments to specify change-controlled deployments

Equally - call back to Mark - docs these days are a lot more human-centric with active voice (Rather than my history in science writeups!)